x-pack/plugins/security/public/management/api_keys/api_keys_grid/api_key_flyout.tsx

```diff
@@ -258,6 +267,7 @@ export const ApiKeyFlyout: FunctionComponent<ApiKeyFlyoutProps> = ({
             >
               <FormField
                 name="name"
+                setTouchedOnBlur={false}
```

Comment:
When the user tabs past the empty name field they used to get instant feedback that the name is required. With this change, this does not occur.
<img width="635" alt="Screenshot 2023-09-29 at 1 15 41 PM" src="https://github.com/elastic/kibana/assets/103939324/81423b21-3437-451b-b710-8bc2800ed52d">

Reply:
@jeramysoucy Good catch! I was investigating this and I've found that whenever a field needs to be validated it loses tab focus incorrectly. Will investigate and fix!

Reply:
@jeramysoucy Created a separate issue to solve this problem here: https://github.com/elastic/kibana/issues/168164

Only solving the form double submit here.